manim/mobject/svg/svg_mobject.py

```diff
@@ -74,6 +74,7 @@ def __init__(
         fill_opacity=1.0,
         should_subdivide_sharp_curves=False,
         should_remove_null_curves=False,
+        style=None,
```

Comment:
@naveen521kk @PhilippImhof I added this because some of the examples for Text/MarkupText in the docs pass in a style parameter (e.g. `style=BOLD`) despite it not being documented. Since both inherit from SVGMobject, I placed it here to prevent TypeErrors; would it be better to place it in their own classes?

Reply:
Thanks for pointing this out. Actually, `MarkupText` should not use `style = BOLD` parameter at all, but rather `weight = BOLD`. Also, after a closer look at the example, I'd say that the text that is currently using it, does not appear in bold.  

The same seems to be true for `Text`. So, unless @naveen521kk has any objections, I will prepare a PR that corrects this error.

Reply:
Alrighty, I'll remove this style parameter from SVGMobject now.